definitions/variable/industry/production.yaml

```diff
@@ -1,14 +1,28 @@
+- Production|Iron and Steel:
+    description: Production of Iron and Steel
+    unit: Mt/yr
+    tier: 1
+    components:
+        - Production|Iron and Steel|{Iron Commodity}
```

Comment:
This replacement doesn't work (if the tag isn't used in the variable name), hence the illegal-character error.

Reply:
OK, so if I want to add a top-level variable like this & add the list of components, would there be another way, or would I always have to list them all, or just not list the components?

Reply:
I would advise against listing components if these are all sub-categories. This is more confusing than helpful.

Reply:
OK, removed.